### ISSUE TITLE
Need to setup cluster using operator having predefined database and user access to the database

### ISSUE THREAD

**Issue by skhilar** [cockroachdb/cockroach-operator#814]
We want to deploy the cluster in k8s using operator. We need to have predefined database other than default one and user with password access to the predefined database. What is the way to configure in operator.

**Comment by goshlanguage** [comment]
I agree, it would be wonderful if we had a way to generate even a user and password from a given k8s secret or otherwise. 

I believe this is already covered here in issue #616 however.

**Comment by Wouter0100** [comment]
A very nice way to do this is by creating dedicated CRD's for this. User, database, and so on. This is way really GitOps friendly.
